spec/requests/linking/unlink_defendant_maat_reference_spec.rb

```diff
@@ -133,8 +136,8 @@
         expect(flash.now[:alert]).to match(/User name must not exceed 10 characters/i)
       end
 
-      it 'redirects to defendant path' do
-        expect(response).to redirect_to edit_defendant_path(id: defendant_id_from_fixture)
+      it 'renders laa_reference path and does not redirect' do
```

Comment:
https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#r466394574

Reply:
https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#r466397250

Reply:
👍 removed and does not redirect
Used expect(response).to render_template('edit')